### ISSUE TITLE
Office error window when moving windows to different virtual desktops

### ISSUE THREAD

**Issue by ghost** [pmb6tz/windows-desktop-switcher#46]
I get this error window when moving any window to different virtual desktop this is super annoying because it steals the window focus, also office applications move really weirdly because this error wants to jump at my face![ApplicationFrameHost_2019-10-05_16-49-40](https://user-images.githubusercontent.com/7440025/66255821-f6986480-e790-11e9-9b11-dba32ee6793d.png)
here's my winver if it even matters here.
![winver_2019-10-05_16-49-02](https://user-images.githubusercontent.com/7440025/66255992-4d526e00-e792-11e9-8984-a91ac7426047.png)

**Comment by Elijas** [comment]
Hi, thanks for submitting an issue. 

Are you moving the non-Office window with the app's hotkey?

 Does this happen when you move a non-Office window manually (Win+Tab)? 

Does moving Office windows themselves trigger the error too?

---

It seems that this bug is known for Microsoft UWP apps
https://social.msdn.microsoft.com/Forums/ru-RU/a451f743-4ad9-42ff-a084-ea5c5508ce65/uwphow-to-fix-xbox-uwp-navigation-error-with-code-unknown12?forum=wpdevelop

**Comment by ghost** [comment]
Yes, I'm moving non-Office and Office windows and it doesn't happen with win+tab.
Moving Office windows triggers this too. Sometimes when moving window from let's say 4 to 3 it goes from 4 to 1 to 3 at this point this has only happened with Office windows but I have a feeling that this happens because the office error window pops on the first desktop

**Comment by Elijas** [comment]
Maybe it's related to #62
